src/vm/arm64/crthelpers.S

```diff
@@ -157,130 +157,123 @@ LEAF_END_MARKED JIT_MemSet, _TEXT
 //    // If not aligned then make it 8-byte aligned   
 //    if(((uintptr_t)dst&0x7) != 0)
 //    {
-//        if(((uintptr_t)dst&0x3) == 0)
+//        if(count == 0)
+//            return
+//        if(dst&0x1)
 //        {
-//            *(UINT*)dst = *(UINT*)src;
-//            dst = (UINT*)dst + 1;
-//            src = (UINT*)src + 1;
-//            count-=4;
+//            *(unit8_t*)dst = *(unit8_t*)src;
+//            dst = (unit8_t*)dst + 1;
+//            src = (unit8_t*)src + 1;
+//            count-=1;
 //        }
-//        else if(((uintptr_t)dst&0x1) == 0)
+//        if(count == 0)
```

Comment:
This looks broken if count is 1 and `(dst&0x2)` is not zero. It will copy two bytes instead of just one.